tests/test_encoder.py

```diff
@@ -138,3 +138,28 @@ def test_ConvVTStage():
     out, cls_tokens = encoder(test_tensor1)
     assert out.shape == torch.Size([32, 64, 56, 56])
     del encoder
+
+
+def test_TubeletEmbedding():
+    from vformer.encoder.embedding.video_patch_embeddings import TubeletEmbedding
```

Comment:
Can you please move this import to the top of the file since we've been doing that for the rest of the test functions?